dev-mode/sbt-plugin/src/main/scala/play/sbt/run/PlayReload.scala

```diff
@@ -38,7 +38,7 @@ object PlayReload {
             .find(_.severity == Severity.Error)
             .map(CompilationException)
             .getOrElse(UnexpectedException(Some("The compilation failed without reporting any problem!"), Some(e)))
-        case e: Exception => UnexpectedException(unexpected = Some(e))
+        case e => UnexpectedException(unexpected = Some(e))
```

Comment:
I will feel a little safer with this instead

```suggestion
        case NonFatal(e) => UnexpectedException(unexpected = Some(e))
```

WDYT?

Reply:
Sorry @jtjeferreira, I think I misled you with the GitHub suggestion. You probably needs to add an import as well.

Reply:
should be fixed!